pkg/webui/console/views/application-payload-formatters/index.js

```diff
@@ -108,10 +110,15 @@ export default class ApplicationPayloadFormatters extends React.Component {
       />
     )
 
+    const applicationFormatterInfo = (
+      <Notification className={style.notification} info content={m.infoText} />
```

Comment:
```suggestion
      <Notification className={style.notification} info content={m.infoText} small />
```
Please set the `small` flag for the link info notification as well.